scripts/enforce-binary-compatibility

```diff
@@ -8,12 +8,9 @@ source "$self_dir"/utils.sh
 # Note: this should be kept in sync with frontendUi/build.gradle.
 # We're not including any 193.* (aka 2019.3) release here since that's the version we're currently building against anyway
 # (see build.gradle -> subprojects.ext.intellijVersion).
-intellij_versions=(192.5728.98)
-# Also tested against 2020.1 EAP (201.6487.11); passes the check successfully.
-# As for now, we're declaring ourselves incompatible with 201.*.
-# 2020.1 is only available as EAP so far anyway, let's wait until it's officially released.
+intellij_versions=(201.6668.121)
```

Comment:
The purpose of the script is to test the versions that we:
1. want to be compatible with AND
1. don't build against

(Not sure if the *.121/*.122 difference here is important, but...) in general we don't need to test against the 201.* IntelliJ builds in this script (since we've just built our plugin against an IntelliJ 201.6668.122). We need to explicitly test binary compatibility with 192.* and 193.*, though - the versions that we do NOT build against.

Reply:
Btw while you're here... pls change the location to which the verifier jar/intellij package are downloaded to sth like ~/.cache/intellij-verifier (`mkdir -p` first ofc)

Downloading into project folder was a bad idea on my part, this files could get accidentally committed etc.

Reply:
ok, so I will revert this to `intellij_versions=(192.5728.98)`
I also will change verifier folder to `./.cache/intellij-verifier` (maybe not in `~`)

Reply:
Revert 192.5728.98... but also include the most recent 193.* release (b/c we want to stay compatible with 2019.3 as well, and we don't build against 2019.3 either).

Reply:
Note that the below condition is **unlikely to be broken** very often in practice...

_For any 3 quarterly releases X, Y, Z where X is earlier than Y and Y earlier than Z_
_and a plugin P,_
_if P is binary compatible with the extreme ones (X and Z)_,
_then P is binary compatible with Y as well_.

but that's **not guaranteed** either, given that each quarterly-ish release of IntelliJ breaks binary compatibility :/

CC @mkondratek

Reply:
In our case substitute X=192.*, Y=193.*, Z=201.*

Reply:
> I also will change verifier folder to ./.cache/intellij-verifier (maybe not in ~)

I'd still go for `~/.cache` since `./.cache` can still get committed accidentally (unless it's git-ignored). `~/.cache` seems a pretty standard place for various pieces of software to put their caches (e.g. I have ~30 subfolders there in my home folder)

Reply:
Also (less imporant), even if it's gitignored/dockerignored etc., it can still accidentally e.g. go into a zip when copying the directory which is ofc easier to spot but still undesired

Reply:
ok, 193.* version added for testing
Regarding to destination folder: ok, I'll change this to `~`